js/modules/bookList.js

```diff
@@ -0,0 +1,80 @@
+export default class BookListing {
+  // declare a constructor function
+  constructor() {
+    if (localStorage.getItem('bookList')) {
+      this.list = JSON.parse(localStorage.getItem('bookList'));
+    } else {
+      this.list = [];
+    }
+  }
+
+  // create a display method
+  displayList() {
```

Comment:
- [ ] Great working on the modules and the design but I highly recommend making the functions to be arrow fucntions to follow ES6 best practices and once it is one of the requirements so kindly update it 💪🏻😊🔥🚀

Reply:
@Reem-lab Thanks for the feedback. However, those are methods in the BookList class. According to the resources given, below is a specimen of a typical ES6 class with a method. My approach is consistence with the given [reference](http://es6-features.org/#ClassDefinition)
![reference](https://user-images.githubusercontent.com/22054464/200671755-074238d4-d6a9-4cd2-97c0-296d8fd6a146.png)